notifications_api_common/management/commands/register_kanalen.py

```diff
@@ -15,6 +18,27 @@ class KanaalExists(Exception):
     pass
 
 
+class KanaalException(Exception):
+    kanaal: str
+    data: dict | list
+
+    def __init__(self, kanaal: str, data: dict | list = {}):
```

Comment:
```suggestion
    def __init__(self, kanaal: str, data: dict | list = None):
```
Please don't use mutable default values